lib/index.js

```diff
@@ -31,15 +31,15 @@ internals.implementation = function (server, options) {
       if (!token && request.auth.mode === 'optional') {
         return reply.continue({ credentials: {} });
       }
-      
+
       if (!token) {
-        return reply(Boom.unauthorized('Missing auth token'));
+        return reply(Boom.unauthorized(null, 'Token'));
```

Comment:
It would be possible, but thats not we want to do and it's outside the point here.
Hapi offers this functionality for authentication plugins and its was supported by the library i want to replace `hapi-auth-jwt` thats why I submitted this PR.